docs/specs/validation.yml

```diff
@@ -340,3 +340,74 @@ outputs:
   .errors.log: |
     ["warning","h1-missing","H1 is required. Use a single hash (#) followed by a space to create your top-level heading.","a.md",1]
 ---
+# Validate included markdown files within Yaml file
+inputs:
+  docfx.yml: |
+    markdownValidationRules: rules.json
+  rules.json: |
+    {
+      "headings": {
+        "name": "Headings",
+        "description": "Validates H1 and other heading content",
+        "aliases": null,
+        "rules": [
+          {
+            "type": "H1Missing",
+            "message": "H1 is required. Use a single hash (#) followed by a space to create your top-level heading.",
+            "exclusions": [ "toc", "includes", "hubpage", "landingpage" ],
+            "severity": "WARNING",
+            "code": "h1-missing"
+          }
+        ]
+      }
+    }
+  docs/a.yml: |
+    #YamlMime:YamlDocument
+    documentType: LandingData
+    title: my title
+    content: |
+      [!include[](b.md)]
+  docs/b.md: |
+    This is test.
+outputs:
+  docs/a.json: |
+    {
+      "title": "my title"
+    }
+  docs/b.json:
+  .errors.log: |
+    ["warning","h1-missing","H1 is required. Use a single hash (#) followed by a space to create your top-level heading.","docs/b.md",1]
+---
+# Skip content validation for yaml files
+inputs:
+  docfx.yml: |
+    markdownValidationRules: rules.json
+  rules.json: |
+    {
+      "headings": {
+        "name": "Headings",
+        "description": "Validates H1 and other heading content",
+        "aliases": null,
+        "rules": [
+          {
+            "type": "H1Missing",
+            "message": "H1 is required. Use a single hash (#) followed by a space to create your top-level heading.",
+            "exclusions": [ "toc", "includes", "hubpage", "landingpage" ],
```

Comment:
Do we want respect this `exclusions` property or blindly exclude all markdown files? If there is no intend to support this property, consider removing it from the test case, as it brings confusion.

Reply:
> Do we want respect this `exclusions` property or blindly exclude all markdown files? If there is no intend to support this property, consider removing it from the test case, as it brings confusion.

Done.